src/api/db/data/20180214132015_project_log_entry_user_name.rb

```diff
@@ -0,0 +1,17 @@
+class ProjectLogEntryUserName < ActiveRecord::Migration[5.1]
+  def up
+    sql = <<-SQL
+      UPDATE project_log_entries
+      INNER JOIN users ON users.id = project_log_entries.user_name
```

Comment:
Have you tried it?
Because while thinking about it, I assume this will go wrong as you probably can't join ``user_name:string(255) `` and ``id:integer`` :cry:

Reply:
nope, I've trusted you :nerd_face: